build-test.sh

```diff
@@ -476,18 +476,15 @@ build_native_projects()
     platformArch="$1"
     intermediatesForBuild="$2"
 
-    extraCmakeArguments="-DCLR_CMAKE_TARGET_OS=${__BuildOS} -DCLR_CMAKE_HOST_ARCH=${platformArch}"
```

Comment:
Aren't the arguments that you have removed needed?

Reply:
`CLR_CMAKE_TARGET_OS` is only used by `pgosupport.cmake`, which we don't use in our test build.

`CLR_CMAKE_HOST_ARCH` is something we re-calculate on Unix platforms in `configurecompiler.cmake`
https://github.com/dotnet/coreclr/blob/580024875e08a1569b847086ac54f3b549fff334/configurecompiler.cmake#L133-L144